site/en/docs/web-platform/conditional-focus/index.md

```diff
@@ -56,11 +57,10 @@ When deciding whether to focus, it is possible to take the [Capture Handle] into
 ```js
 // Retain focus if capturing a tab dialed to example.com.
 // Focus anything else.
-const displaySurface = track.getSettings().displaySurface;
 const origin = track.getCaptureHandle().origin;
 if (displaySurface == "browser" && origin == "https://example.com") {
   controller.setFocusBehavior("no-focus-change");
-} else {
+} else if (displaySurface != "screen") {
```

Comment:
Should be "monitor". See [here](https://www.w3.org/TR/screen-capture/#idl-def-DisplayCaptureSurfaceType.monitor). (But I think comparing to "window" is more future-proof.)

Reply:
Fixed.